### DISCUSSION TITLE
App Insights Connection String in the browser code

### DISCUSSION THREAD

**Discussion by Gareth064** [IvanJosipovic/BlazorApplicationInsights#312] [Q&A] [answered]
Hey. This is a bit random possibly, but should there be any concerns about the connection string to our app insights being viewable or obtained from the Browser?

Could someone, in theory, take the connection string and start sending logs to our app insights?

**Comment by maisak** [comment, marked as answer]
[Documentation](https://learn.microsoft.com/en-us/azure/azure-monitor/app/sdk-connection-string?tabs=js#do-new-azure-regions-require-the-use-of-connection-strings) says it is kinda hard/impossible to protect on client side, so you have to live with it.

![image](https://github.com/user-attachments/assets/f01b6cce-0776-4b2d-b7ca-bddd20d71102)

**Reply by Gareth064** [reply]
Cheers @maisak
